kuryr_kubernetes/controller/managers/prometheus_exporter.py

```diff
@@ -117,8 +121,24 @@ def _record_ports_quota_per_subnet_metric(self):
             ports_availability = total_num_addresses-ports_count
             self.port_quota_per_subnet.labels(**labels).set(ports_availability)
 
+    def _record_lbs_metrics(self):
+        """Records the number of members available per LB and the LB state"""
+        critical_lb_names = ['openshift-dns/dns-default', 'default/kubernetes']
+        lbaas_spec = {}
+        self._drv_lbaas.add_tags('loadbalancer', lbaas_spec)
+        lbs = self._os_lb.load_balancers(**lbaas_spec)
```

Comment:
Would it be possible to query for KLBs first (K8s guarantees that names are unique) and then use IDs to identify which LBs we care about?